test/test_manifest.json

```diff
@@ -1507,6 +1507,15 @@
        "rounds": 1,
        "type": "load"
     },
+    {  "id": "issue13487",
+       "file": "pdfs/issue13487.pdf",
+       "md5": "80044ae1ce3fb8c748d6a72f868d5de1",
+       "link": true,
+       "firstPage": 7,
+       "lastPage": 8,
```

Comment:
The firstPage/lastPage should be the *same* number to limit the test to only one page.